src/components/Dropdown/DropdownOption.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, CSSProperties, ReactNode, useEffect } from 'react';
+
+export interface IDropdownOptionProps {
+  /** children must be React Element */
+  children?: ReactNode;
+  /** set customized css class */
+  className?: string;
+  /** set customized css style */
+  cssStyle?: CSSProperties;
```

Comment:
if cssStyle is just the same as style, you could just use style

Reply:
Ok, I will change it to style ;)